dist/profile/manifests/ssl-cert.pp

```diff
@@ -0,0 +1,29 @@
+#
+# SSL certificates
+#
+class profile::ssl-cert (
+  # all injected from hiera
+  $id,        # identify which private key / certificate pair we should use. This usually comes from hieradata/clients/*.yaml
```

Comment:
I'm not sure if I understand the question. They reference specific files and entry in hiera.
In dbd4d30 I've expanded this comment to hopefully clarify this.